src/api/firebase.js

```diff
@@ -55,6 +54,72 @@ export function getItemData(snapshot) {
 		.filter((item) => item.name !== null);
 }
 
+function sortItems(item1, item2) {
+	if (item1.daysUntilPurchase < item2.daysUntilPurchase) {
+		return -1;
+	} else if (item1.daysUntilPurchase > item2.daysUntilPurchase) {
+		return 1;
+	}
+	if (item1.name < item2.name) {
```

Comment:
Should we normalize the item name like we do on the add item page? What happens when we sort `['bAby', 'aBBy', 'Baby', 'Abby']` all with the same date?

Reply:
Hey Lyle, I don't think we need to worry about the scenario you bought up. Remember how for issue #9 me and Katherine would prevent duplicate items like that ending up in our list in the first place.

Reply:
@Yaosaur @jeremiahfallin there seems to be an issue with the alphabetical sorting based on case:
<img width="571" alt="image" src="https://github.com/the-collab-lab/tcl-55-smart-shopping-list/assets/77761206/848a1ad1-af0e-48e3-9e67-dbd09a0582ac">

Reply:
@mentalcaries good catch, should be fixed now.